controller/controller.go

```diff
@@ -177,23 +177,40 @@ func (c *controller) updateIngresses() (err error) {
 		}
 	}()
 
+	// Get ingresses
 	var ingresses []*v1beta1.Ingress
+
 	if c.namespaceSelector == nil {
 		ingresses, err = c.client.GetAllIngresses()
 	} else {
 		ingresses, err = c.client.GetIngresses(c.namespaceSelector)
 	}
+
 	log.Infof("Found %d ingresses", len(ingresses))
```

Comment:
Can we perhaps move this further down, so once it's checked that the length of `ingresses` and `services` is > 0, it will log as one line? e.g. `log.Infof("Found %d Ingresses and %d services", len(ingresses), len(services))`

At the moment we log what it's found, then we log the reason if it errors (the same message). If we keep the individual logs, can we change them to `log.Debugf()` just to reduce the noise? 
```
Found 0 ingresses
Unable to update ingresses: found 0 ingresses
```

```suggestion
	log.Debugf("Found %d ingresses", len(ingresses))

	if err != nil {
		return err
	}

	if len(ingresses) == 0 {
		return errors.New("found 0 ingresses")
	}

	// Get services
	services, err := c.client.GetServices()

	if err != nil {
		return err
	}

	log.Debugf("Found %d services", len(services))

	if len(services) == 0 {
		return errors.New("found 0 services")
	}

	log.Infof("Found %d Ingresses and %d services", len(ingresses), len(services))
```